packages/strapi-helper-plugin/lib/src/components/LeftMenuList/index.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { isEmpty } from 'lodash';
+import matchSorter from 'match-sorter';
+
+import Wrapper from './Wrapper';
+import List from './List';
+import Search from './Search';
+
+import LeftMenuLink from '../LeftMenuLink';
+import LeftMenuSubList from '../LeftMenuSubList';
+
+function LeftMenuList({ customLink, links, title }) {
+  const [search, setSearch] = useState('');
+  const [showSearch, setShowSearch] = useState(false);
+
+  const { Component, componentProps } = customLink;
```

Comment:
```
const { Component, componentProps } = customLink || {
  Component: null,
  componentProps: {},
 };
```

This way we can use without passing a `customLink` props equals to `{}` or without this props.